apps/web/src/services/alert-manager/v2/components/ServiceList.vue

```diff
@@ -234,6 +236,16 @@ onMounted(async () => {
     await fetchBothLists();
 });
 
+// ✅ event listener to change healthy page size (6 -> 8) when window width is 1920px or more
+let prevHealthyPageSize = serviceListPageStore.healthyPageSize;
+useEventListener(window, 'resize', () => {
+    serviceListPageStore.updateHealthyPageSizeByWindowWidth();
+    if (serviceListPageStore.healthyPageSize !== prevHealthyPageSize) {
+        prevHealthyPageSize = serviceListPageStore.healthyPageSize;
+        fetchHealthyServiceList();
+    }
+});
+
```

Comment:
Suggestion-1: How about considering debounced resizing? 
Suggestion-2: How about using `setHealthyPageSize` with condition written in this file(not store)? (access of `window` object) because I think using `window` object in store will make it difficult to understand code in this context.